src/components/attendees/AttendeeTable.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import AttendeeRow from './AttendeeRow';
+import { Attendee } from '../../interfaces/Attendee';
+import { ReactComponent as FilterIcon } from '../../assets/Attendees/filter.svg';
+import { ReactComponent as SortIcon } from '../../assets/Attendees/sort.svg';
+import './styles.css';
+import Logistics from '../../pages/logistics';
+
+/**
+ * Creation of the Attendee Table component for the Attendee page. The filter for the Attendee Table will be applied here to display the corresponding hackers.
+ * @returns the table of selected hackathon attendees.
+ */
+export default function AttendeeTable() {
+	const attendeeData = [];
+
+	const [search, setSearch] = useState('');
+	const [namesAscending, setNamesAscending] = useState(false);
+
+	let person1: Attendee = {
+		firstName: 'Jane Jane',
+		lastName: 'Doe',
+		email: 'email@domain.com',
+		discord: 'discord#1234',
+		oAuth: 'email',
+		school: 'University of Central Florida',
+		isAccepted: false,
+		isConfirmed: true,
+		isCheckedIn: true,
+		userLink: 'https://google.com',
+		id: '1',
+		pronouns: {
+			subjective: 'she/her',
+			objective: 'she/her',
+		},
+		phoneNumber: 'num',
+		age: 12,
+		role: 'attendee',
+	};
+	// Remove after adding APIs
+	for (var i = 0; i < 10; i++) {
+		attendeeData.push({
```

Comment:
This doesn't matter right now since we're just using this for testing purposes, but for future reference this can be destructured as:

```
attendeeData.push({
	...person1,
	id: '' + i,
});
```

Reply:
updated!